_posts/en/newsletters/2018-08-21-newsletter.md

```diff
@@ -0,0 +1,172 @@
+  measurements by Naumenko).  Maxwell is also working on making it
+  possible for a newly-started (or long-disconnected) node to
+  efficiently sync its mempool with its peers using this same basic
+  mechanism, reducing the requirement for small independent miners to
+  maintain expensive high-uptime and well-connected infrastructure.
+
+- **Dandelion protocol DoS-resistant stem routing** being worked on
+  by Suhas Daftuar.  The [Dandelion protocol][] is expected to make
+  it extremely difficult for an adversary to determine the IP
+  address of any program that creates a Bitcoin transaction (even if
+  they don't use Tor), but the new method of handling unconfirmed
+  transactions privately for a time during the "stem" phase has to
+  be secured against attacks that could waste node bandwidth and
+  memory.
+
+For additional details, please see the [conversation log][2018-08-16
+meeting log].
+
+## Notable commits
+
+*Notable commits this week in [Bitcoin Core][core commits], [LND][lnd
+commits], and [C-lightning][cl commits].*
+
+{% comment %}<!-- IMO, c-lightning only had 6 commits this week, mostly
+minor doc updates, so no news for them.  I'm still leaving them
+mentioned above for easy copy/paste next week. -harding -->{% endcomment %}
+
+{% include linkers/github-log.md 
+  refname="core commits"
+  repo="bitcoin/bitcoin"
+  start="1b04b55f2d22078ca79cd38fc1078e15fa9cbe94"
+  end="df660aa7717a6f4784e90535a13a95d82244565a"
+%}
+{% include linkers/github-log.md 
+  refname="lnd commits"
+  repo="lightningnetwork/lnd"
+  start="6989316b11c51922b4c6ae3507ac06680ec530b9"
+  end="3f5ec993300e38369110706ac83301b8875500d6"
+%}
+{% include linkers/github-log.md 
+  refname="cl commits"
+  repo="ElementsProject/lightning"
+  start="a97955845ff43d4780b33a7301695db33823c57c"
+  end="80a875a9a54e26c2ea4c90aee8fe606ddcc27c55"
+%}
+
+- Bitcoin Core 0.17 branched: this allows developers to focus on
+  ensuring stability, translation completeness, and other release
+  features on that branch while development of new features continues on
+  the master branch.  This Notable Commits section only focuses on the
+  master development branch of each project, so commits mentioned from
+  this point forward are much less likely to be included in the 0.17
+  version of Bitcoin Core and should not be expected before the 0.18
+  version.
+
+- [Bitcoin Core #13917][] and [Bitcoin Core #13960][] improve the
+  [BIP174][] Partially-Signed Bitcoin Transaction (PBST) handling in
+  ambiguous situations.
+
+- [Bitcoin Core #11526][] makes it much easier to build Bitcoin Core
+  with Microsoft Visual Studio 2017, including being able to use the Visual
+  Studio debugger.
+
+- [Bitcoin Core #13918][] provides the 10th, 25th, 50th, 75th, and 90th
+  percentile feerates for a historic block with the `getblockstats` RPC
+  introduced to the master development branch a couple months ago.
+
+- [LND #1693][] allows LND's autopilot funding mechanism to optionally
```

Comment:
As discussed, the primary motivation for this PR is to allow autopilot to open many channels in parallel, since it doesn't need to wait for one channel to be confirmed before using the change to open the next channel.